conf/test_2_fasta_pathogen.config

```diff
@@ -5,25 +5,25 @@
     Defines input files and everything required to run a fast and simple pipeline test.
 
     Use as follows:
-        nextflow run nf-core/dualrnaseq -profile test_hackathon,<docker/singularity> --outdir <OUTDIR>
+        nextflow run nf-core/dualrnaseq -profile test_2_fasta_pathogen,<docker/singularity> --outdir <OUTDIR>
```

Comment:
I agree the naming should be different. This naming doesn't quite make sense as the fasta contains host and pathogen reads.
Perhaps test_small as the files are small and were designed on purpose for fast testing run-times